plugins/flytekit-onnx-scikitlearn/tests/test_onnx_scikitlearn.py

```diff
@@ -0,0 +1,112 @@
+from typing import Annotated, List, NamedTuple, TypeVar
+
+import numpy
+import onnxruntime as rt
+import pandas as pd
+from flytekitplugins.onnxscikitlearn import ScikitLearn2ONNX, ScikitLearn2ONNXConfig
+from skl2onnx.common._apply_operation import apply_mul
+from skl2onnx.common.data_types import FloatTensorType
+from skl2onnx.proto import onnx_proto
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.datasets import load_iris
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+from flytekit import task, workflow
+from flytekit.types.file.file import FlyteFile
+
+
+def test_onnx_scikitlearn_simple():
+    TrainOutput = NamedTuple(
+        "TrainOutput",
+        [
+            (
+                "model",
+                Annotated[
+                    ScikitLearn2ONNX,
+                    ScikitLearn2ONNXConfig(
+                        initial_types=[("float_input", FloatTensorType([None, 4]))],
+                        target_opset=12,
+                    ),
+                ],
+            ),
+            ("test", pd.DataFrame),
+        ],
+    )
+
+    @task
+    def train() -> TrainOutput:
+        iris = load_iris(as_frame=True)
+        X, y = iris.data, iris.target
+        X_train, X_test, y_train, _ = train_test_split(X, y)
+        model = RandomForestClassifier()
+        model.fit(X_train, y_train)
+
+        return TrainOutput(test=X_test, model=ScikitLearn2ONNX(model))
+
+    @task
+    def predict(
+        model: FlyteFile[TypeVar("onnx")],
```

Comment:
I think we can add `TypeVar("onnx")` to this [file](https://github.com/flyteorg/flytekit/blob/master/flytekit/types/file/__init__.py), and here we can update to `model: ONNXFile`